crates/turbopack-ecmascript/src/analyzer/mod.rs

```diff
@@ -169,8 +248,7 @@ impl Display for ConstantValue {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
             ConstantValue::Undefined => write!(f, "undefined"),
-            ConstantValue::StrWord(str) => write!(f, "\"{str}\""),
-            ConstantValue::StrAtom(str) => write!(f, "\"{str}\""),
+            ConstantValue::Str(str) => write!(f, "\"{str}\""),
```

Comment:
Should this use `stringify_js` instead? Although I don't expect we'll ever want to attempt to evaluate the result of this.